src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1332,4 +1332,17 @@ private function getErrorMessages(): array
             CannotUpdateProductException::class => $this->trans('An error occurred while updating the status for an object.', 'Admin.Notifications.Error'),
         ];
     }
+
+    /**
+     * @return array
+     */
+    private function getLogDataContext($id_product = null, $error_code = null, $allow_duplicate = null): array
+    {
+        return [
+                'object_type' => 'Product',
+                'object_id' => $id_product,
+                'error_code' => $error_code,
+                'allow_duplicate' => $allow_duplicate,
+            ];
```

Comment:
Something is broken with php-cs-fixer...

Reply:
Sorry @PierreRambaud , but ... who should i trust, you or `php-cs-fixer` 😅 ?